skfuzzy/control/fuzzyvariable.py

```diff
@@ -11,6 +14,41 @@
     from .ordereddict import OrderedDict
 
 
+class FuzzyVariableAdjective(object):
```

Comment:
This does much more than replace the `self.active` structure.  It's trying to mimic how fuzzy variables are defined in IEC 61131-7 (the fuzzy control language).  They implement a variable like this:
```
    FUZZIFY Variable1
        TERM Bad := (0, 0) (0, 1) (50, 0) ; (* aka the adjective Variable1['Bad'] *)
        TERM Normal := (14, 0) (50, 1) (83, 0) ;
        TERM Good := (50, 0) (100, 1) (100, 0) ;
    END_FUZZIFY
```
Regarding the name adjective, I stole it from pyfuzzy's implementation, as that is what they call a "TERM."  To me it makes a lot of sense, but I guess we can change it.

http://pyfuzzy.sourceforge.net/doc/class_diagram.png